deal_thief/models/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import (
+    Index,
+    Date,
+    Column,
+    ForeignKey,
+    Integer,
+    Unicode,
+    UnicodeText
+)
+from sqlalchemy.orm import relationship
+from .meta import Base
+from passlib.apps import custom_app_context as pwd_context
+from datetime import datetime
+
+
+class User(Base):
+    """User schema."""
+
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    first_name = Column(Unicode(255), nullable=False)
+    last_name = Column(Unicode(255), nullable=False)
+    email = Column(Unicode(255), nullable=False, unique=True)
+    password = Column(Unicode(255), nullable=False)
+    sign_up_date = Column(Date())
```

Comment:
did you notice that SQLAlchemy columns have the ability to add defaults that are set automatically when an object is first created?  See [the documentation](http://docs.sqlalchemy.org/en/latest/core/defaults.html) for more info.